JavaComparator/Comparator.java

```diff
@@ -0,0 +1,133 @@
+import java.util.ArrayList;
+
+public class Comparator {
+    String file_orig, file_tripped;
```

Comment:
In java, variable names should be of this form variableName (they will be of form variable_name only in Python). Please change all the variable names in all java files to reflect this.